setup.cfg

```diff
@@ -28,13 +29,18 @@ zip_safe = False
 install_requires =
     invenio-base>=1.2.5
     invenio-i18n>=1.3.1
+    invenio-records-resources>=1.0.0,<2.0.0
```

Comment:
major: we should keep dependencies for this module to a minimum, since it's going to be a dependency of any module that needs to send out notifications, and we want to avoid (indirect) circular dependencies.

I think for now we'll probably just need `invenio-mail` (since we're sending mails with it) and `invenio-celery` (for the tasks definition).

Reply:
it's imported here https://github.com/inveniosoftware/invenio-notifications/pull/7/files#diff-3cb3d81840449f3418b6c2f67ef6f91edd25a88da88d570ac18ee1aa8b545255R13-R21

We could get away without the mixin and the `FromConfig` though.